src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -269,6 +274,7 @@ static bool addTriggeredSubscriptions
       subs.insert(std::pair<string, TriggeredSubscription*>(subIdStr, trigs));
     }
   }
+  releaseMongoConnection(connection);
```

Comment:
Fixed in 5974d5f4658d64823329d6122c2a4a49fdfda636... before I discovered that kill() is not avalabile in legacy-1.0.2, but in a newer driver version :(

Thus, I have prepared the code, so just removing a #if will make the code work with kill (see 2971d46538f8801f4468f2d8d0d288b872e6f261)